redis/cache/redispipebp/init_test.go

```diff
@@ -53,7 +60,9 @@ func TestMain(m *testing.M) {
 	client = redisx.BaseSync{
```

Comment:
this value of `client` is no longer used by anything?

Reply:
Removed.